examples/medplum-fhircast-demo/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "module": "ESNext",
+    "skipLibCheck": true,
+
+    /* Bundler mode */
+    "moduleResolution": "bundler",
+    "allowImportingTsExtensions": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx",
+
+    /* Linting */
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true
+  },
+  "include": ["src"],
+  "references": [{ "path": "./tsconfig.node.json" }]
```

Comment:
I think we can merge `tsconfig.json` and `tsconfig.node.json` into a single file, yeah?  I don't know why Vite does this by default.

Reply:
I guess it's just for `vite.config.ts` since that runs in Node. Probably doesn't matter if your Vite config is small though since you're probably not doing a whole lot. 

I guess DOM vs Node env.